components/automate-ui/src/app/entities/userperms/userperms.effects.ts

```diff
@@ -47,4 +50,10 @@ export class UserPermEffects {
     mergeMap((action: GetUserParamPerms) => this.requests.fetchParameterized(action.payload).pipe(
       map((resp: UserPermsResponsePayload) => new GetUserParamPermsSuccess(resp.endpoints)),
       catchError((error: HttpErrorResponse) => of(new GetUserParamPermsFailure(error))))));
+
+  @Effect()
+  fetchApplyRulesStatusAllowed$ = interval(1000 * POLLING_INTERVAL_IN_SECONDS).pipe(
+    mergeMap(() => this.requests.fetchSome({ paths: [ProjectConstants.APPLY_RULES_ENDPOINT] }).pipe(
+      map((resp: UserPermsResponsePayload) => new GetSomeUserPermsSuccess(resp.endpoints)),
+      catchError((error: HttpErrorResponse) => of(new GetSomeUserPermsFailure(error))))));
 }
```

Comment:
In order to make the go/no-go decision on checking-rule-status we need the specific permission already in the store. This periodically updates that data.
(Elsewhere there is a similar block of code already in place to periodically update the version.)

Reply:
I believe these kinds of blocks are what got us to introduce `waitForAngular(false)` in our `automate-ui` "e2e" tests. (Back when we did the periodical license status check.) As a consequence, writing tests that aren't flakey has become more difficult.

Do you think spreading these is OK, since we're not that invested into those tests anymore, or should we perhaps revisit the alternatives?

Reply:
Interesting point... I suspect the "damage" was done by the first `interval` call, so adding more would not make it worse. 🤷‍♂ But probably not a big concern since, as you say, we're likely moving away from protractor towards Cypress.

Reply:
> so adding more would not make it worse. 

Debatable! The amount of work required to fix this is increased. 📈